docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,25 @@
+# Release process and schedule
+
+The release manager is responsible for ensuring that the following steps are performed:
```

Comment:
We should add a step where a new release manager is selected. I'd say this should happen directly after a release whereas the current release manager is responsible for all point releases until the next release.